MorphicServer.Tests/EndpointTests.cs

```diff
@@ -33,6 +33,7 @@
 using Microsoft.AspNetCore.TestHost;
 using Microsoft.Extensions.Configuration;
 using Serilog;
+using Serilog.Formatting.Compact;
```

Comment:
Note sure why it's showing this change as part of your PR. I merged it into master, and then into this branch. A normal 'git diff master' doesn't show it. Very strange.

Reply:
Oh I see. I merged in master, but this was branched off of the doc branch. Hm.